easybuild/easyconfigs/o/OpenMPI/OpenMPI-5.0.3-GCC-13.2.0.eb

```diff
@@ -0,0 +1,43 @@
+name = 'OpenMPI'
+version = '5.0.3'
+
+homepage = 'https://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-3 implementation."""
+
+toolchain = {'name': 'GCC', 'version': '13.2.0'}
```

Comment:
I'll bump this to `13.3.0`, so it can be used in upcoming `foss/2024a`...

Reply:
Bump to `13.3.0` done in https://github.com/boegel/easybuild-easyconfigs/pull/96